homeworks/sasha.demidenko_SashaDemi/GamesProject/index.html

```diff
@@ -51,5 +51,6 @@ <h1 class="about_title">About me</h1>
                 </div>
             </div>
         </div>
+        <script src="scripts/script.js"></script>
```

Comment:
Now your login form isn't working, proly you should change it back. And rename the script to `login-validation.js` or some other obvious name

Reply:
You could remove the connection of this script here, cause it is useless and cause an error 
![image](https://github.com/MastersAcademy/fe-course-basic-2023/assets/98031768/f326f786-6a42-47b3-a54e-19ca008a0a0d)